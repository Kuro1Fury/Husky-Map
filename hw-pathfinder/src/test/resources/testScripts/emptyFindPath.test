# Find a path in an empty graph

CreateGraph A

FindPath A n1 n2