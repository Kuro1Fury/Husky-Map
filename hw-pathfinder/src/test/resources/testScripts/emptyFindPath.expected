# Find a path in an empty graph

created graph A

unknown: n1
unknown: n2